README.md

```diff
@@ -2,16 +2,20 @@
 
 # react-number-format
 
-React Number format is a input formatter library with a sophisticated and lightweight caret engine.
+React Number Format is an input-formatter library with a sophisticated and light weight caret engine.  It ensures that a user can only enter text that meets specific numeric or string patterns, and formats the input value for display.
 
 ### Features
 
-1. Prefix, suffix and thousand separator.
-2. Custom pattern formatting.
-3. Masking.
-4. Custom formatting handler.
-5. Format number in an input or format as a simple text.
-6. Fully customizable
+1. Prefix, suffix and thousands separator.
+1. Input Masking.
+1. Format number in an input or format as a simple text.
+1. Custom pattern formatting.
+1. Custom formatting handler.
+1. Fully customizable
```

Comment:
All are changed to 1.

Reply:
Which is valid markup.  The benefit of doing it this way is that you can resort or add new lines without having to renumber everything.

Reply:
Ohh yes, actually.